WeakAuras/GenericTrigger.lua

```diff
@@ -3105,6 +3106,7 @@ do
       bar.spellId = tostring(spellId)
       bar.text = text
       bar.duration = duration
+      bar.bwIsCd = cd
```

Comment:
I think `bwIsCd` is not good name for this field, i'd suggest `isCooldown` instead
From bigwigs source i see value could be true/false/nil, can you make the value only boolean?
Can you add it as a filter for the trigger, and a also as a condition?